nativelib/src/main/resources/scala-native/gc/immix/ImmixGC.c

```diff
@@ -11,12 +11,18 @@
 #include "WeakReferences.h"
 #include "Settings.h"
 #include "shared/Parsing.h"
+#include "shared/Time.h"
 #ifdef SCALANATIVE_MULTITHREADING_ENABLED
 #include "immix_commix/Synchronizer.h"
 #endif
 #include "MutatorThread.h"
 #include <stdatomic.h>
 
+// The total (accumulated) number of GC runs
+static size_t GC_STATS_COLLECTION_TOTAL = 0L;
+// The total (accumulated) elapsed time in nanos of GC runs
+static size_t GC_STATS_COLLECTION_DURATION_TOTAL = 0L;
+
```

Comment:
Have you considered make these part of perhaps a shared `jmx.c` unit so they can be shared across GCs instead of duplicated? That might leave a space for future enhancements to go.

Reply:
Good point, thanks for the suggestion. I added `jmx.h` and `jmx.c`.